### ISSUE TITLE
Change Remand Reasons to support AMA appeals

### ISSUE THREAD

**Issue by laurjpeterson** [department-of-veterans-affairs/caseflow#5760]
When DAS is deprecated, we can implement the new, streamlined remand reasons / duty to assist errors under AMA. When attorneys and judges decide an issue on appeal should be remanded (legacy language) or contains an error (AMA language), they need to add a reason for data capture.

All AMA remands should be due to an RO issue, not a Board issue, so we do not need a "pre-certification" or "post-certification" designation like for legacy appeals.

## AC
- A Remand Reasons page exists for AMA appeals
- If an attorney chooses the issue disposition Remand, they navigate to this Remand Reasons page
- The AMA Remand Reasons design is exactly the same as for legacy, but with different reasons/copy as below:
    - Duty to notify
         - No Notice Sent 
             - Pre or post AOJ decision
         - Incorrect Notice Sent
             - Pre or post AOJ decision
         - Legally Inadequate Notice
            - Pre or post AOJ decision
   - Duty to assist
      - VA records
         - Pre or post AOJ decision
      - Private records
         - Pre or post AOJ decision
      - Service personnel records
         - Pre or post AOJ decision 
      - Service treatment records
         - Pre or post AOJ decision 
   - Other government records
         - Pre or post AOJ decision 
- Medical Examinations and Opinions
   - Medical examinations
      - Pre or post AOJ decision 
   - Medical opinions
      - Pre or post AOJ decision
   - Advisory medical opinion
      - Pre or post AOJ decision

#### Out of scope for now
- Every issue with a remand reason creates a corresponding DTA error EP [TO BE CONFIRMED]:

Remand Reason Category| Remand Reason | EP
---|---|---
Duty to Notify | No Notice Sent |040BDE	         Board DTA Error
Duty to Notify | Incorrect Notice Sent  |040BDE	         Board DTA Error
Duty to Notify | Legally Inadequate Notice  |040BDE	         Board DTA Error
Duty to Assist | VA records |040BDE	         Board DTA Error
Duty to Assist | Private records |040BDE	         Board DTA Error
Duty to Assist | Service personnel records |040BDE	         Board DTA Error
Duty to Assist | Service treatment records |040BDE	         Board DTA Error
Duty to Assist | Other government records |040BDE	         Board DTA Error
Medical Exams and Opinions | Medical examinations |040BDE	         Board DTA Error
Medical Exams and Opinions | Medical opinions | ??? 040BDEIMO	Board DTA Error with IMO
Medical Exams and Opinions | Advisory medical opinions | ??? 040BDEIMO	Board DTA Error with IMO

---

## Open Questions
- What EPs will be created for remands (medical opinion, duty to notify)?
   - Can we use legacy EPs?
- How should DTA errors work in relation to appeals? What EPs will be created for DTA errors?
- For legacy appeals, users designate if the remand reason was pre-certification or post-certification. In AMA world, we should also ask the user (attorney) to designate if the remand reason was before the appeal got to the Board or after. **What language should we use?**

## Resources
Email exchange with Emily Tamlyn on April 23, 2018 explained that remands for duty to notify will be allowed in the new system

> Update: Remands for the duty to notify will be allowed in the new system
> The Board and VBA had taken the initial position that remands for the duty to notify were not allowed by AMA and that this was part of the design of the new system. During the concurrence process, OGC disagreed with this interpretation and approach. The language of the draft regulation was altered; the preamble as excerpted below was changed. While the Board would still anticipate that such remands would be rare, we will want to continue to internally track such remands (as we currently are now) in the future. As noted below, however, Section 5 is not concerned with such tracking.

- [Attorney checkout crosswalk](https://docs.google.com/presentation/d/1bpXEmP_0Dpan4EbDpU7_UniEhAIeMfuPlvW5YaAoUuU/edit)
 

- Re: Rachel 6/18
   - I talked to AMO Jennifer about BEAAM remands and post-decision actions. She said they have procedures in place to process DTA errors identified on HLR. They already have issued 433 of these and have readjudicated 24. Jennifer said the higher-level adjudicator drafts a deferred rating using form 21-6789 and creates a 683 EP with a claim label that says something like "HLR DTA Error". She said she would look into getting the Board a special claim label for the 683 EP. If any of our BEAAM decisions identify a DTA error, we would issue the decision and email the AMO-Ops email box separately to alert them of the remand.

- [Slack discussion](https://dsva.slack.com/archives/CA1QBDLEL/p1530033940000767) about non-DTA error "remands".

- [AMA reporting requirements](https://github.com/department-of-veterans-affairs/appeals-pm/blob/master/VAIMA-Implementation/reporting-breakdown.md)

### Consolidated AMA + legacy remand reasons
- Duty to notify
   - No Notice Sent 
      - Pre or post certification (legacy)
      - Pre or post AOJ decision (AMA)
   - Incorrect Notice Sent
      - Pre or post certification (legacy)
      - Pre or post AOJ decision (AMA)
   - Legally Inadequate Notice
      - Pre or post certification (legacy)
      - Pre or post AOJ decision (AMA)
- Due process (legacy only)
      - Pre or post certification (legacy)
- Duty to assist
   - VA records
      - Pre or post certification (legacy)
      - Pre or post AOJ decision (AMA)
   - Private records
      - Pre or post certification (legacy)
      - Pre or post AOJ decision (AMA)
   - Service personnel records
      - Pre or post certification (legacy)
      - Pre or post AOJ decision (AMA)
   - Service treatment records
      - Pre or post certification (legacy)
      - Pre or post AOJ decision (AMA)
   - Other government records
      - Pre or post certification (legacy)
      - Pre or post AOJ decision (AMA)
- Medical Examinations and Opinions
   - Medical examinations
      - Pre or post certification (legacy)
      - Pre or post AOJ decision (AMA)
   - Medical opinions
      - Pre or post certification (legacy)
      - Pre or post AOJ decision (AMA)
   - Advisory medical opinion
      - Pre or post certification (legacy)
      - Pre or post AOJ decision (AMA)

**Comment by laurjpeterson** [comment]
- Remand is an issue disposition for issue.
- Remand of type DTA error should create a Supplemental Claim.
   - This applies to Higher Level Reviews and Appeals (Hearing, Evidence Submission, Direct Review)
- Caseflow Intake creates Supplemental Claim.
- Caseflow Intake then creates the EP for VBA to process the Supplemental Claim in VBMS.


---
- How should we connect a Supplemental Claim to the appeal?

**Comment by aroltsch** [comment]
Where to store issues dispositions and remand reasons.

### Option 1.
HLR and SCs will have issue dispositions and remand reasons on request issues. Appeals will have issue dispositions and remand reasons on decision issues and the final disposition will live on the request issue. Appeals final disposition can be calculated based on the combination of the issue dispositions of decision issues

### Option 2.
HLR, SCs  and Appeals will have issue dispositions and remand reasons on decision issues. The final disposition will live on the request issue (final_disposition). 


### Option 3
HLR, SCs  and Appeals will have issue dispositions and remand reasons on decision issues. The final disposition for appeals will be calculated at runtime based on the combination of the issue dispositions and will not be stored at the database level.


How remand reasons should be stored?
1. An array of strings 
2. Clear text
3. Separate table that will reference either request issue or decision issue

**Comment by shanear** [comment]
As far as remand reasons, I believe VBMS is going to bake them into the dispositions. I think we may want to do the same for consistency.

**Comment by laurjpeterson** [comment]
Excerpt from email with Emily Tamlyn, Legislation, Regulation, and Policy, 8/17:

§ 20.802, Remand for Correction of Error.

a) Remand. Unless the issue or issues can be granted in full, the Board shall remand the appeal to the agency of original jurisdiction for correction of an error on the part of the agency of original jurisdiction to satisfy its duties under 38 U.S.C. 5103A, if the error occurred prior to the date of the agency of original jurisdiction decision on appeal. **The Board may remand for correction of any other error by the agency of original jurisdiction in satisfying a regulatory or statutory duty, if correction of the error would have a reasonable possibility of aiding in substantiating the appellant's claim.** The remand must specify the action to be taken by the agency of original jurisdiction.

(b) Advisory Medical Opinion. If the Board determines that an error as described in paragraph (a) of this section may only be corrected by obtaining an advisory medical opinion from a medical expert who is not an employee of the Department of Veterans Affairs, the Board shall remand the case to the agency of original jurisdiction to obtain such an opinion, specifying the questions to be posed to the independent medical expert providing the advisory medical opinion.

(c) Action by agency of original jurisdiction after receipt of remand. After correction of any error identified in the Board's remand, the agency of original jurisdiction must readjudicate the claim and provide notice of the decision under 38 U.S.C. 5104, to include notice under 38 U.S.C. 5104C of a claimant's options for further review of the agency of original jurisdiction's decision. The agency of original jurisdiction must provide for the expeditious treatment of any claim that is remanded by the Board.

(Authority: 38 U.S.C. 5103A, 5109, 5109B, 7102, 7104(a), 7105)

Sadia also points out this portion of the preamble, which discusses this rule: “Additionally, the Board may remand for the correction of any other error by the agency of original jurisdiction in satisfying a regulatory or statutory duty where there is a reasonable possibility that correction of the error would aid in substantiating the claim, but need not remand solely for correction of a procedural defect as this would be inconsistent with the statutory framework of Public Law 115-55.”

My recollection (supported by my email archive of back and forth between us, OGC and VBA) is that the highlighted portion above was our best compromise for a catchall. We all agreed, this should be a rare error.

In sum, I do think you should raise with VBMS that they need to capture this as a reason for error, at least for the Board, if not HLR.

**Comment by nicholasholtz** [comment]
Reached out to Mike Palmer last week. Will follow-up on Wednesday, 8/29

**Comment by laurjpeterson** [comment]
Email from Emily Tamlyn, LRP, this morning about issue dispositions and remand reasons

---
Good morning,

 

We met with Dustin Ware yesterday to discuss Section 5, the spreadsheet you sent us regarding Section 5 and IT requirements, and your mural.

 

We agreed that for legacy cases, the pre and post cert remand and decision reasons need to stay tracked. Even though there will not be a form 8, there will be some electronic signaling of transfer of the legacy case from VBA to the Board. Attorneys will need to be informed of what the transfer is and then log the remand reason accordingly as they do now.  
We agreed that for AMA cases, it would be advisable to use pre and post AOJ decision for both remand reasons and decisions. We tracked this to Section 5 (1)(U) of the law.
Dustin was concerned that the mural does not exactly specify that the following remand reasons must be tracked: advisory medical opinions under 38 USC 5109(d), medical opinions (i.e., NOT those under 5109(d)), and medical examinations.
 

Please let us know if you have questions about the above. We need to meet once more next week to finish going through the concerns on the second spreadsheet. I will send you a summary after that meeting as well, if necessary.

 

Thanks!

 

Emily Tamlyn, Counsel

Office of Legislation, Regulations & Policy

Board of Veterans’ Appeals

202.632.5617

**Comment by dannysellers** [comment]
#### Remand reasons to be used in both cases
(codes listed from [`ACTIVE_REMAND_REASONS_BY_ID.json`](https://github.com/department-of-veterans-affairs/caseflow/blob/master/client/constants/ACTIVE_REMAND_REASONS_BY_ID.json))
- Duty to notify
   - No Notice Sent (`"AA"`)
   - Incorrect Notice Sent (`"AB"`)
   - Legally Inadequate Notice (`"AC"`)
- Duty to assist
   - VA records (`"BD"`)
   - Private records (`"BF"`)
   - Service personnel records (`"BB"`)
   - Service treatment records (`"BA"`)
   - Other government records (`"BG"`?)
- Medical Examinations and Opinions
   - Medical examinations (?)
   - Medical opinions (?)
   - Advisory medical opinion (?)

Two questions:
- There are some inexact matches between existing remand reasons and this list. For example, the three remand reasons under Medical Examinations.
- For legacy remand reasons, we send a VACOLS ID (i.e. `"AB"`) to the server. Should we send the same IDs for AMA reasons?

**Comment by laurjpeterson** [comment]
> There are some inexact matches between existing remand reasons and this list. For example, the three remand reasons under Medical Examinations.
- yes, this is because there are different, and more scoped down remand reasons for AMA. 

> For legacy remand reasons, we send a VACOLS ID (i.e. "AB") to the server. Should we send the same IDs for AMA reasons?
- we shouldn't send anything to VACOLS for AMA appeals

**Comment by dannysellers** [comment]
Ok, so we have two similar but different sets of codes, that works.

> We shouldn't send anything to VACOLS for AMA appeals

Right, but we will need to send a code / id to the server indicating which remand reason(s) the user has selected. I meant, should we use the same codes as VACOLS wants to indicate the new reasons (though the data would only be written to the Caseflow database)? For example, should `"BB"` correspond to "Service personnel records" for both legacy and ama appeals?

I guess by virtue of us using different reasons, we might as well have different ids for them, in which case, I'd defer to backend people for what those codes should look like. I'll just start with a parameterized version of the text (i.e. `"medical_examinations"`). cc @aroltsch @shanear

**Comment by shanear** [comment]
There is only one type of Supplemental Claim EP made for all DTA/Remand/etc so we really don't care what the reason was when we make the follow up Supplemental Claim.

**Comment by shanear** [comment]
@laurjpeterson 

We need confirmation on this before we can close this story out
<img width="829" alt="screen shot 2018-10-10 at 5 15 10 pm" src="https://user-images.githubusercontent.com/1596259/46766490-2319bd80-ccb0-11e8-96f1-a4d610b321bf.png">

**Comment by laurjpeterson** [comment]
Emailed AMO 10/16 to confirm the 040BDEIMO Board DTA Error with IMO

**Comment by laurjpeterson** [comment]
Confirmed mapping with AMO and LRP 10/16

`independent medical opinion` = `advisory medical opinion`, and is separate from medical opinions.

only issues with the following characteristics should result in a `040BDEIMO Board DTA Error with IMO` EP:
- disposition - remand
- remand reason category - `medical exams and opinions`
- remand reason - `advisory medical opinion`

The `040BDEIMOPMC PMC Board DTA Error - w/IMO` EP must have the same remand reason of `advisory medical opinion` but additionally, must also be a pension issue.

**Comment by laurjpeterson** [comment]
See Mural with updated, approved issue disposition, remand reason (if applicable), EP, and supplemental claim mapping - @shanear 

https://app.mural.co/t/workqueue2001/m/workqueue2001/1534279378866/d4531cbe67b6690068a6bd052d4deed55f6ab131

**Comment by shanear** [comment]
awesome. That's all i needed to close this one out.
